app/src/androidTest/java/ly/loud/loudly/util/database/KeyTest.java

```diff
@@ -0,0 +1,109 @@
+package ly.loud.loudly.util.database;
+
+import android.support.annotation.NonNull;
+import android.support.test.runner.AndroidJUnit4;
+import com.pushtorefresh.storio.sqlite.StorIOSQLite;
+import ly.loud.loudly.application.Loudly;
+import ly.loud.loudly.new_base.KeyKeeper;
+import ly.loud.loudly.new_base.Networks;
+import ly.loud.loudly.util.Equality;
+import ly.loud.loudly.test.Generators;
+import ly.loud.loudly.util.database.entities.Key;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+/**
+ * @author Danil Kolikov
+ */
+@RunWith(AndroidJUnit4.class)
+public class KeyTest {
+    private StorIOSQLite keysDatabase;
+    private Random random;
+
+    private static void checkKeys(@NonNull List<KeyKeeper> keys) {
+        for (int i = 0; i < Networks.NETWORK_COUNT; i++) {
+            KeyKeeper saved = Loudly.getContext().getKeyKeeper(i);
```

Comment:
Now it works like it. Anyway, now we often use Loudly.getContext(). Until we get rid of it, I think it's acceptable to use it for keys